client/lib/analytics/ad-tracking/google-analytics-4.ts

```diff
@@ -28,6 +31,9 @@ export function setup( params: Gtag.ConfigParams ) {
 	if ( isAkismetCheckout() ) {
 		window.gtag( 'config', TRACKING_IDS.akismetGoogleGA4Gtag, params );
 	}
+	if ( isA8CForAgencies() ) {
```

Comment:
Nitpick not related to tracking specifically, could we unify how we call A4A/A8C For Agencies in (analytics) code?

We can call this function isA4A() or other occurrences, to A8C_FOR_AGENCIES - just consistency could increase readability in the long term.

Reply:
Good point. I've made some adjustment to apply your suggestion 👍 

Thanks for the feedback.